sideboard/lib/_threads.py

```diff
@@ -91,8 +94,9 @@ def run(self):
             except:
                 log.error('unexpected error', exc_info=True)
 
-            if self.interval:
-                self.stopped.wait(self.interval)
+            interval = config['thread_wait_interval'] if self.interval is None else self.interval
+            if interval:
+                self.stopped.wait(interval)
```

Comment:
Not in this case, because ``0`` is falsey and someone may have deliberately specified an interval of ``0``.  In that case we would NOT want to fall back on the default ``thread_wait_interval`` value.